dijet/tasks/jer.py

```diff
@@ -60,28 +60,45 @@ def output(self) -> dict[law.FileSystemTarget]:
         return outp
 
     def run(self):
+        # load extrapolation results
         results_extrapolation = self.load_extrapolation()
 
-        # ### Now JER SM Data
-        jer = results_extrapolation["intercepts"].copy()
-        view = jer.view()
+        # get extrapolated distribution widths
+        h_widths = results_extrapolation["intercepts"]
 
-        # Get index of method to change methods individually
-        # hist.view() only works if the full histgram is taken w.o. selecting the categories before hand
+        # get index on `category` axis corresponding to
+        # the two computation methods
         category_id = {"sm": 1, "fe": 2}
-        # Get list of categories with the correct order
-        categories = list(jer.axes["category"])
+        categories = list(h_widths.axes["category"])
         index_methods = {m: categories.index(category_id[m]) for m in category_id}
 
-        # calcuate jer for standard method
-        view.value[index_methods["sm"]] = view.value[index_methods["sm"], :, :] * np.sqrt(2)
+        # calcuate JER for standard method
+        jer_sm_val = h_widths[index_methods["sm"], :, :].view().value * np.sqrt(2)
+        jer_sm_err = np.sqrt(h_widths[index_methods["sm"], :, :].view().variance) * np.sqrt(2)
```

Comment:
Minor thing, but one could use the shorter `.values()` instead of `view().vaule` here. Same for `variances()`.
```suggestion
        jer_sm_val = h_widths[index_methods["sm"], :, :].values() * np.sqrt(2)
        jer_sm_err = np.sqrt(h_widths[index_methods["sm"], :, :].variances()) * np.sqrt(2)
```